spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -217,6 +224,13 @@ trait CommonJobExecution extends ProjectMetadata {
     })
   }
 
+  protected def addCustomDataToInfoFile(data: Map[String, String]): Unit = {
+    data.foreach { entry =>
+      log.info(s"writing $entry to info file.") // todo debug or remove?
+      Atum.setAdditionalInfo(entry)
```

Comment:
Here, the key/value is added as-is. However, do we want it like this? I have noticed that in `additionalInfo`, standardization-related keys are prefixed with `std_`, raw prefixed with `raw_`, ... so do we want a prefix here, too?

Reply:
I have added `std_ds_` prefix for the dataset properties.

Reply:
Redone to be configurable with the default of empty prefix. See below.